.gitignore

```diff
@@ -8,3 +8,4 @@ logs
 reports
 jmeter.log
 upgrades
+metadata.json
```

Comment:
who produces this file?

Reply:
build.sh

Reply:
why doesn't it remove it when done?

Reply:
its cached so it does not hat every extension json every time